zdm/analyze_cube.py

```diff
@@ -135,10 +143,12 @@ def get_bayesian_data(lls:np.ndarray,
             
             wthemax=np.nanmax(wlls)
             OKlls=np.isfinite(lls) & (lls > themax-3)
-            OKwlls=np.isfinite(lls) & (wlls > wthemax-3)
+            OKwlls=np.isfinite(wlls) & (wlls > wthemax-3)
             
             vector[iv]=np.sum(10**lls[OKlls])
```

Comment:
we should double-check at some point the weighting, to ensure it applies a weighting appropriate to alpha=1 (the best-fit result from J-P depends on the alpha interpretation)